### DISCUSSION TITLE
Add support for the “Justify” alignment option in the `AlignFeature` of the LexicalEditor.

### DISCUSSION THREAD

**Discussion by ju4n97** [payloadcms/payload#4035] [Feature Requests & Ideas]
The `AlignFeature` in the `lexicalEditor` adapter currently supports “Left”, “Center”, and “Right” alignment options, as mentioned in this [Discord question](https://discord.com/channels/967097582721572934/1166770904865460224). It would be beneficial to include a “Justify” alignment option in payload side, as there are scenarios where this feature is needed.

![image](https://github.com/payloadcms/payload/assets/64181348/4535760c-f84a-493e-a45f-23acf46682ff)

**Comment by tallosim** [comment]
Completed and merged with #4868.
